src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((li) => {
+    const text = li.firstChild.nodeValue.trim();
```

Comment:
Ensure that `li.firstChild` is a text node. If `li.firstChild` is not a text node, `nodeValue` will be `null`, and calling `trim()` will cause an error. Consider checking if `li.firstChild` is a text node before accessing `nodeValue`.